lib/rubocop/cop/rails/migration_class_name.rb

```diff
@@ -41,7 +41,7 @@ def on_class(node)
         def basename_without_timestamp
           filepath = processed_source.file_path
           basename = File.basename(filepath, '.rb')
-          basename.sub(/\A\d+_/, '')
+          basename.sub(/\A\d+_/, '').sub(/\..+\z/, '')
```

Comment:
Can you use `delete_suffix` instead?

```suggestion
          basename.sub(/\A\d+_/, '').delete_suffix('.active_storage.rb')
```

Reply:
I think this cop should be more generic than just cater for `active_storage` since this format `.{gem_name}.rb` is a pattern that will most probably be used in the future by other Rails modules or other gems that generate migrations for your app.

Reply:
Ah, I get it. In that case, extracting to a private method will make it easier to read and will be better. e.g.:

```ruby
# e.g.: from `add_blobs.active_storage.rb` to `add_blobs`.
def remove_gem_rb_suffix(file_name)
  file_name.sub(/\..+\z/, '') 
end
```

Reply:
Good idea! I’ve rebased with your suggestion \o/